lib/options.js

```diff
@@ -40,8 +41,18 @@ function normalizeOptions(options) {
 	options.karma = options.karma || karma;
 	options.lintGlobs = options.lintGlobs || codeGlobs;
 	options.uglifySrc = options.uglifySrc || 'build/**/*.js';
-	options.moduleName = options.moduleName || 'metal';
+	options.moduleName = options.moduleName || packageJSON.name || 'metal';
```

Comment:
I'm afraid of what changes may this trigger down the line in the `moduleName` usage in amd, for example... I'd keep it like it was, since it already has a default. If someone is using it and wants to have a proper name, it can be specified already, right?.